backend/de.metas.payment.sepa/base/src/main/java/de/metas/payment/sepa/api/impl/CreateSEPAExportFromPaySelectionCommand.java

```diff
@@ -56,24 +68,29 @@ class CreateSEPAExportFromPaySelectionCommand
 	private static final AdMessageKey ERR_C_BP_BankAccount_BankNotSet = AdMessageKey.of("de.metas.payment.sepa.C_BP_BankAccount_BankNotSet");
 	private static final AdMessageKey ERR_C_BP_BankAccount_SEPA_CreditorIdentifierNotSet = AdMessageKey.of("de.metas.payment.sepa.C_BP_BankAccount_SEPA_CreditorIdentifierNotSet");
 	private static final AdMessageKey ERR_C_Bank_SwiftCodeNotSet = AdMessageKey.of("de.metas.payment.sepa.C_Bank_SwiftCodeNotSet");
+	private static final AdMessageKey ERR_SEPA_Invoice_Nor_Payment_Set = AdMessageKey.of("SEPA_Invoice_Nor_Payment_Set");
+	private static final AdMessageKey MSG_SEPA_Payment_Refund = AdMessageKey.of("SEPA_Payment_Refund");
 
-
-	private final IPaySelectionDAO paySelectionRepo = Services.get(IPaySelectionDAO.class);
+	private final IPaySelectionBL paySelectionBL = Services.get(IPaySelectionBL.class);
 	private final IBPartnerOrgBL partnerOrgBL = Services.get(IBPartnerOrgBL.class);
 	private final BankRepository bankRepo = SpringContextHolder.instance.getBean(BankRepository.class);
+	private final IPaymentBL paymentBL = Services.get(IPaymentBL.class);
+	private final IMsgBL msgBL = Services.get(IMsgBL.class);
+	private final IInvoiceBL invoiceBL = Services.get(IInvoiceBL.class);
+	private final IBPBankAccountDAO bankAccountDAO = Services.get(IBPBankAccountDAO.class);
 
 	private final I_C_PaySelection source;
 
-	public CreateSEPAExportFromPaySelectionCommand(@NonNull final org.compiere.model.I_C_PaySelection source)
+	public CreateSEPAExportFromPaySelectionCommand(@NonNull final I_C_PaySelection source)
 	{
-		this.source = InterfaceWrapperHelper.create(source, I_C_PaySelection.class);
+		this.source = create(source, I_C_PaySelection.class);
```

Comment:
i think calling `create` is no longer needed. u can use it directly, i.e. this.source = source;